proto/p2p/p2p.proto

```diff
@@ -459,6 +458,13 @@ message ProtocolMetadata {
   bytes prev = 5;
 }
 
+message EmptyVoteMetadata {
```

Comment:
`ProtocolMetadata` includes a `version` field. It seems to me like we should either remove the `version` field from `ProtocolMetadata` or we should include it here.

fwiw I've always felt like the `version` field was weird...

Reply:
> since we might want to extend the protocol metadata later on

I don't disagree that we may want to change the protocol metadata later on, but I think that any changes to the metadata should probably be based on the conventional ways to update proto messages. Rather than with the explicit version field.

That being said, (since in my preferred world the protocol metadata wouldn't have the version) I don't have a problem merging the empty vote metadata as is (without the version).